test/functional/transactions_tests.js

```diff
@@ -110,7 +110,7 @@ describe('Transactions (spec)', function() {
         afterEach(() => cleanupAfterSuite(testContext));
 
         testSuite.tests.forEach(testData => {
-          const maybeSkipIt = testData.skipReason ? it.skip : it;
+          const maybeSkipIt = testData.skipReason || testSuite.name === 'pin-mongos' ? it.skip : it;
```

Comment:
Generally I'd say we should factor this out into something more extensible (check an array of potentially skipped tests, for examples), but since we're likely to remove this soon for scheduled work I think this is fine. What do you think @daprahamian?

Reply:
I think it is fine for now.